gkr_iop/src/precompiles/lookup_keccakf.rs

```diff
@@ -1106,6 +843,15 @@ pub fn run_faster_keccakf<E: ExtensionField>(
     };
     exit_span!(span);
 
+    // mock prover
+    let out_wits = gkr_output
```

Comment:
Not related to this line but same files:

unittest `test_keccakf_nonpow2` currently are with `#[ignore]` tag, I think we can remove the tag so it can be run normally

(this unittest also passed)

Reply:
this part will affect performance quite a lot. suggest to move check to non-release build as below
```diff
-    // mock prover
-    let out_wits = gkr_output
-        .0
-        .iter()
-        .map(|poly| poly.evaluations())
-        .collect_vec();
-    MockProver::check(gkr_circuit.clone(), &gkr_witness, out_wits, vec![])
-        .expect("mock prover failed");
+    if cfg!(debug_assertions) {
+        // mock prover
+        let out_wits = gkr_output
+            .0
+            .iter()
+            .map(|poly| poly.evaluations())
+            .collect_vec();
+        MockProver::check(gkr_circuit.clone(), &gkr_witness, out_wits, vec![])
+            .expect("mock prover failed");
+    }
```

Reply:
Updated in https://github.com/scroll-tech/ceno/pull/970/commits/7dfa0c95960b8b3d652ba9d3e1c2e40bbb0d04e4